environment_dev.yaml

```
@@ -7,19 +7,26 @@ dependencies:
 - python=3.9
 - numpy
 - pandas
+- sphinx
 - xarray
```

```suggestion
- xarray
- dask
- netCDF4
- bottleneck
```

Adding xarray's recommended dependencies